README.md

```diff
@@ -727,6 +731,26 @@ func main() {
 	do(true)
 }
 ```
+## Type Assertions
+A type assertion provides access to an interface value's underlying concrete value.
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+	var i interface{} = "hello"
+
+	//The statement asserts that varible value i is of type string
+	s, ok := i.(string)
+
+	//To test whether an interface value holds a specific type, a type assertion can 	 
+  // return two values: the underlying value and a boolean value that reports whether
+	// the assertion succeeded.
+	fmt.Println(s, ok)
+}
```

Comment:
good one, however i don't think it's worth 15 LoC to explain it.

Reply:
Thanks for feedback! I have added concise example of type assertion.